### PR TITLE
op-guide: reorganize and update migration overview

### PR DESCRIPTION
**Pull request by lilin90**
Via: https://github.com/pingcap/docs/pull/1038

@dcalvin PTAL

### CODE REVIEW

File: op-guide/migration-overview.md [hunk 1]
```diff
@@ -5,49 +5,47 @@ category: advanced
 
 # 数据迁移概述
 
-## 概述
+本文档详细介绍了如何将数据从 MySQL 或 CSV 数据源迁移到 TiDB。
```
Comment (dcalvin) [inline op-guide/migration-overview.md]: 感觉是概述，不是"详细”介绍
```suggestion
本文介绍了如何将数据从 MySQL 或 CSV 数据源迁移到 TiDB。
```

File: op-guide/migration-overview.md [hunk 2]
```diff
@@ -5,49 +5,47 @@ category: advanced
...
 
-这里我们假定 MySQL 以及 TiDB 服务信息如下：
+在上述数据迁移过程中，会用到如下工具：
 
-|Name|Address|Port|User|Password|
-|----|-------|----|----|--------|
-|MySQL|127.0.0.1|3306|root|*|
-|TiDB|127.0.0.1|4000|root|*|
+- [Mydumper](/tools/mydumper.md)：用于从 MySQL 导出数据。建议使用 Mydumper，而非 mysqldump。
+- [Loader](/tools/loader.md)：用于将数据导入到 TiDB，支持 Mydumper 的输出数据文件。
```
Comment (dcalvin) [inline op-guide/migration-overview.md]:
```suggestion
- [Loader](/tools/loader.md)：用于将 Mydumper 导出格式的数据导入到 TiDB。
```

File: op-guide/migration-overview.md (hunk 1 above)
Comment (lilin90) [reply op-guide/migration-overview.md]: Exactly. That's the previous wording. I've updated this intro sentence.

File: op-guide/migration-overview.md [hunk 3]
```diff
@@ -5,49 +5,47 @@ category: advanced
...
 # 数据迁移概述
 
-## 概述
+本文档介绍了 TiDB 提供的数据迁移工具，以及不同迁移场景下如何选择迁移工具，从而将数据从 MySQL 或 CSV 数据源迁移到 TiDB。
 
-该文档详细介绍了如何将 MySQL 的数据迁移到 TiDB。
+## 迁移工具
 
-这里我们假定 MySQL 以及 TiDB 服务信息如下：
+在上述数据迁移过程中，会用到如下工具：
```
Comment (dcalvin) [inline op-guide/migration-overview.md]:
```suggestion
在上述数据迁移过程中会用到如下工具：
```

**Pull request merged**